src/ucm/mmap/install.c

```diff
@@ -137,17 +150,23 @@ static ucs_status_t ucs_mmap_install_reloc(int events)
             continue;
         }
 
-        ucm_debug("mmap: installing relocation table entry for %s = %p for event 0x%x",
-                  entry->patch.symbol, entry->patch.value, entry->event_type);
-
-        status = ucm_reloc_modify(&entry->patch);
-        if (status != UCS_OK) {
-            ucm_warn("failed to install relocation table entry for '%s'",
-                     entry->patch.symbol);
-            return status;
+        if (UCM_IS_HOOK_ENABLED(entry)) {
+            ucm_debug("mmap: installing relocation table entry for %s = %p for event 0x%x",
```

Comment:
lets also print the type of hook (bistro/reloc)

Reply:
updated message text